src/file_loader.rs

```diff
@@ -0,0 +1,89 @@
+// Copyright 2013-2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use glib::object::IsA;
+use glib::translate::*;
+use gtk_source_sys;
+use std::boxed::Box;
+use std::ptr;
+use FileLoader;
+
+impl FileLoader {
+    pub fn load_async<
```

Comment:
This should be in a `FileLoaderExtManual` trait so that it can also work on all subclasses. It seems like it's not a final class, or is it?

Reply:
As you can probably tell, I'm a novice at Rust, so will have to be guided by your wisdom here.

Reply:
I don't know the sourceview API so I can't tell you the answer either :) @GuillaumeGomez ?

Reply:
I don't know it either. XD

Reply:
Short look at the headers makes it seem they're not final, so needs a trait.